docs/examples/arrow_example.ipynb

```diff
@@ -295,12 +354,14 @@
    "source": [
     "### Use the power-grid-model\n",
     "\n",
-    "For more extensive examples, visit the [power-grid-model documentation](https://power-grid-model.readthedocs.io/en/stable/index.html)."
+    "The `output_component_types` argument is set to `ComponentAttributeFilterOptions.relevant` to given out columnar data.\n",
```

Comment:
I think that the other comment was about making sure the output is always columnar to avoid again additional copies.